vdb-bench-assembly/plugins/vdb-bench-dataservice/dsEditController.js

```diff
@@ -17,8 +17,20 @@
         
         vm.svcSourcesLoading = SvcSourceSelectionService.isLoading();
         vm.svcSources = SvcSourceSelectionService.getServiceSources();
-        vm.saveButtonDisabled = true;
-        
+        vm.initialSourceName = null;
+        vm.initialSourceTableName = null;
+        vm.sourceInit = true;
+        vm.tableInit = true;
+
+        /*
+         * Set initial source selection
+         */
+        angular.element(document).ready(function () {
+            // Initialize the selections if possible for the selected dataservice
```

Comment:
Apparently, it is better to inject the document rather than using the global variable. It allows unit testing for example. See http://stackoverflow.com/questions/18646756/how-to-run-function-in-angular-controller-on-document-ready for details.
Will be converted to $document.ready(function(){...})